labs/07_troubleshooting_ops.md

```diff
@@ -54,45 +54,43 @@ KUBERNETES_PORT_53_TCP=tcp://172.30.0.1:53
 ...
 ```
 
-## Logfiles anschauen
+## View log files
 
-Die Logfiles zu einem Pod können sowohl in der Web Console als auch auch im CLI angezeigt werden.
+The log files for a pod can be displayed in the web console as well as in the cli:
 
 ```
 $ oc logs [POD]
 ```
-Der Parameter `-f` bewirkt analoges Verhalten wie `tail -f`
-
-Befindet sich ein Pod im Status **CrashLoopBackOff** bedeutet dies, dass er auch nach wiederholtem Restarten nicht erfolgreich gestartet werden konnte. Die Logfiles können auch wenn der Pod nicht läuft mit dem folgenden Befehl angezeigt werden.
+The `-f` parameter has the same behavior as `tail -f`
+If a pod has the status **CrashLoopBackOff** this means it could not be started succesfully even after repeated restarts. The logfiles still can be displayed even if the pod is not running with the following command:
 
  ```
 $ oc logs -p [POD]
 ```
 
-Mit OpenShift wird ein EFK (Elasticsearch, Fluentd, Kibana) Stack mitgeliefert, der sämtliche Logfiles sammelt, rotiert und aggregiert. Kibana erlaubt es Logs zu durchsuchen, zu filtern und grafisch aufzubereiten. Weitere Informationen und ein optionales LAB finden sie [hier](../additional-labs/logging_efk_stack.md).
-
+With Openshift an EFK (Elasticsearch, Fluentd, Kibana) stack is delivered, which collects, rotates and aggregates all log files. Kibana allows logs to be searched, filtered and graphically edited. For more information and an optional Lab see [here](../additional-labs/logging_efk_stack.md).
 
 ## Task: LAB7.3 Port Forwarding
 
-OpenShift 3 erlaubt es, beliebige Ports von der Entwicklungs-Workstation auf einen Pod weiterzuleiten. Dies ist z.B. nützlich, um auf Administrationskonsolen, Datenbanken, usw. zuzugreifen, die nicht gegen das Internet exponiert werden und auch sonst nicht erreichbar sind. Im Gegensatz zu OpenShift 2 werden die Portweiterleitungen über dieselbe HTTPS-Verbindung getunnelt, die der OpenShift Client (oc) auch sonst benutzt. Dies erlaubt es auch dann auf OpenShift 3 Platformen zuzugreifen, wenn sich restriktive Firewalls und/oder Proxies zwischen Workstation und OpenShift befinden.
+OpenShift 3 allows us to forward any port from our workstation to the pod. This is usefull to use adminstrationconsoles, databases and so on which should not be exposed towards the internet. In contrast to Openshift 2 the portforwarding is tunneled through the same HTTPS-connection as the Openshift Client (`oc`) uses. This is usefull if there are restrictive Firewalls and/or Proxies between your Workstation and Openshift.
```

Comment:
```suggestion
OpenShift 3 allows us to forward any port from our workstation to the pod. This is useful to use admin-consoles, databases etc. which should not be exposed towards the internet. In contrast to Openshift 2 the portforwarding is tunneled through the same HTTPS-connection as the Openshift Client (`oc`) uses. This is useful if there are restrictive firewalls and/or proxies between your workstation and Openshift.
```

Reply:
Unless "admin-consoles" is a technical term (i.e. if it is plain English) "admin consoles" should be written as two separate words.

Reply:
I believe, there should be a comma before "which". Source: [Grammarly](https://www.grammarly.com/blog/comma-before-which/)

Reply:
"OpenShift" should be written with a capital S letter (because this is the brand name). x2

Reply:
Also, write "OpenShift client" with a small initial letter ("client" is just a normal English word, isn't it?).

Reply:
@bittner I updated my review